v4-client-js/src/clients/native.ts

```diff
@@ -988,3 +1017,54 @@ export async function getMarketPrice(
     return wrappedError(e);
   }
 }
+export async function getNobleBalance(): Promise<String> {
+  try {
+    const client = await getNobleClient();
+    if (client === undefined) {
+      throw new UserError(
+        'client is not connected.',
+      );
+    }
+    if (globalThis.nobleWallet?.address === undefined) {
+      throw new UserError('wallet is not set. Call connectWallet() first');
+    }
+    const coin = await client.getAccountBalance(globalThis.nobleWallet.address, 'uusdc');
+    return encodeJson(coin);
+  } catch (error) {
+    return wrappedError(error);
+  }
+}
+
+export async function sendNobleIBC(squidPayload: string): Promise<String> {
```

Comment:
does this only accept squidPayloads or does it except generic payloads as well?

Reply:
it does specifically have to be a squid IBC payload as I parse the json here with that expectation. Similar to `withdrawToIBC` in this file